docs/references/nextjs/auth-middleware.mdx

```diff
@@ -79,7 +143,7 @@ export const config = {
 };
 ```
 
-### Making pages public using `publicRoutes`
+### Controlling public routes using `publicRoutes`
```

Comment:
I liked "Make pages public..." best... Is there a reason to use "control" instead?

Reply:
I changed it to 'Making routes public...' I can use 'pages' but I think 'routes' is more accurate.